### PR TITLE
Add Toodledo [Other]

### PR DESCRIPTION
**Pull request by offsky**
Toodledo is a suite of productivity tools to help you manage your to-do list, notes, habits and more

### CODE REVIEW

File: _data/other.yml [hunk 1]
```diff
@@ -335,3 +335,10 @@ websites:
       software: Yes
       sms: Yes
       doc: https://support.zendesk.com/hc/en-us/articles/203824246
+
+    - name: Toodledo
+      url: http://www.toodledo.com
```
Comment (Jawshy) [inline _data/other.yml]: Their site is available through HTTPS which 2factorauth prefers.

File: _data/other.yml [hunk 2]
```diff
@@ -335,3 +335,10 @@ websites:
       software: Yes
       sms: Yes
       doc: https://support.zendesk.com/hc/en-us/articles/203824246
+
+    - name: Toodledo
+      url: https://www.toodledo.com
+      img: toodledo.png
+      tfa: Yes
+      software: Yes
+      doc: https://www.toodledo.com/otp.php
```
Comment (stephengroat) [inline _data/other.yml]: for the `doc`, what about https://www.toodledo.com/info/help.php#168? It's not behind a sign-in page

**Comment by Carlgo11** [comment]
Merge conflict :disappointed:

**Review by stephengroat — CHANGES_REQUESTED** [review]
Because of additions to the `other.yml` file, the name is now out of alphabetical order. could you merge and reorder it?

**Comment by offsky** [comment]
@stephengroat I have merged and reordered it so its alphabetically again.

**Review by stephengroat — APPROVED** [review]

**Comment by Carlgo11** [comment]
Sorry for the delay @offsky. Your pull request is now merged and will show up on our site soon.

**Pull request merged**